components/asic/asic.c

```diff
@@ -1,55 +1,53 @@
 #include <string.h>
-
 #include <esp_log.h>
-
 #include "bm1397.h"
 #include "bm1366.h"
 #include "bm1368.h"
 #include "bm1370.h"
-
 #include "asic.h"
 #include "device_config.h"
+#include "power_management_task.h"
```

Comment:
damn this sucks. I know you didn't cause the problem and this change is just exposing it more but asic shouldnt care at all about the tasks that are leveraging it. we really got away from that.

Reply:
there will more come up. its not the first time im doing such things^^